packages/network/src/logic/node/StreamManager.ts

```diff
@@ -22,7 +22,7 @@ function keyForDetector({ publisherId, msgChainId }: MessageLayer.MessageID) {
 export class StreamManager {
     private readonly streams: Map<SPIDKey,StreamState> = new Map<SPIDKey,StreamState>()
 
-    setUpStream(spid: SPID, oneDirectional = false): void {
+    setUpStream(spid: SPID, isBehindProxy = false): void {
```

Comment:
Given that the value of `isBehindProxy` is rather consequential and that this is internal logic, I would advocate not using a default value here. But it's a matter of taste, so I'll leave it up to you.
```suggestion
    setUpStream(spid: SPID, isBehindProxy: boolean): void {
```